File: app/src/main/java/com/neome/java/api/home/drawer/msg/MsgUserNotificationList.java
// neome.ai API - do not change
//

package com.neome.java.api.home.drawer.msg;

import com.neome.java.api.core.base.msg.MsgVersion;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class MsgUserNotificationList extends MsgVersion
{
  @Nullable
  public Long limit;

  @Nullable
  public Long offset;
}

File: app/src/main/java/com/neome/api/home/drawer/msg/MsgUserNotificationList.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.home.drawer.msg

import com.neome.api.core.base.msg.MsgVersion

open class MsgUserNotificationList : MsgVersion()
{
  var limit: Number? = null
  var offset: Number? = null
}